superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx

```diff
@@ -67,7 +72,7 @@ const RunQueryActionButton = ({
         cta
         onClick={() => runQuery(true)}
         key="run-async-btn"
-        tooltip={t('Run query asynchronously (Ctrl + ↵)')}
+        tooltip={t('Run query (Ctrl + Return)')}
```

Comment:
```suggestion
		tooltip={t('Run Query (Ctrl + Return)')}
```

Reply:
The design has `Query` as lowercase, but let's check with Stephen!